crates/storage/trie/nibble.rs

```diff
@@ -322,7 +322,7 @@ impl NibbleVec {
         }
     }
 
-    #[cfg(test)]
+    //#[cfg(test)]
```

Comment:
Why not remove it instead of commenting it out?

Reply:
I think this was leftover for when I was trying out stuff, I should restore it, ty for catching it!